backend/kafka_backend.go

```diff
@@ -0,0 +1,114 @@
+package backend
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/skroutz/downloader/job"
+)
+
+var eventsWG sync.WaitGroup
+
+// KafkaBackend notifies about a job completion by producing to a Kafka topic.
+type KafkaBackend struct {
+	producer *kafka.Producer
+	reports  chan job.CallbackInfo
+}
+
+// ID returns the identifier code for the respective backend
+// which is "kafka".
```

Comment:
A simpler:

> ID returns "kafka"

should be sufficient. The "identifier code for the respective backend" part is already documented in the Backend interface and should not be necessary here too.

Ditto for `httpBackend.ID()`